docs/core/core.adoc

```diff
@@ -30,14 +30,11 @@ Refer to the link:./getting-started[Getting Started] guide for more information
 
 == Usage
 
-The Core module supports parsing of the configuration file generated by the mobile CLI, in practice this feature is used internally by other modules of the SDK however:
+The Core module supports parsing of the configuration file generated by the mobile CLI, in practice this feature is used internally by other modules of the SDK:
```

Comment:
Don't like the `however`, @wtrocki ? :P

Reply:
Very funny, **however** we need to be adults here :D